src/applications/vaos/referral-appointments/components/PendingReferralCard.unit.spec.jsx

```diff
@@ -43,11 +43,4 @@ describe('VAOS Component: PendingReferralCard', () => {
       ),
     ).to.exist;
   });
-  // TODO: figure out how to test the click event
```

Comment:
I removed this test because this web component is something we maintain so we shouldn't be testing it's functionality.